README.md

```diff
@@ -9,72 +9,105 @@ Gulp build tasks for use across Fozzie modules.
-    ```
+build(gulp, /*options*/);
+```
 
-    Then add a `.babelrc` file, with the `babel-preset-es2015` preset, to the root of your project:
+You can optionally [pass in options which will override the default config values](#options).
 
-    ```javascript
-    {
-        "presets": ["es2015"]
-    }
-    ```
+That's it! You can now run any of [the Gulp tasks](#the-gulp-tasks).
 
-    If you do not add a `.babelrc` file (you may be writing es5 code for example) then the code will be bundled up as is.
+### Optional setup
 
-1. Add an `.eslintrc` file to the root of your project. The recommended eslint configuration for fozzie modules can be found in [`/fozzie-config/.eslintrc`](https://github.com/justeat/gulp-build-fozzie/fozzie-config/.eslintrc) – simply add the contents of this into your projects `.eslintrc` file to use the JS linting rules we recommend when running the [`scripts:lint`](#scriptslint) task.
+#### Transpile es2015 code
 
-1. To use our recommended fozzie stylelint linting rules, add the following into your `package.json` file:
+To ensure that the [`scripts:bundle`](#scriptsbundle) task can transpile es2015 code, add the `babel-preset-es2015` preset to the project:
 
-    ```
-    "stylelint": {
-        "extends": "@justeat/stylelint-config-fozzie"
-    }
-    ```
+```bash
+yarn add babel-preset-es2015
+```
 
-    If you wish to extend these rules, you can also define your own `.stylelintrc` file in the root of your project with the rules that you wish to override.
+Then add a `.babelrc` file, with the `babel-preset-es2015` preset, to the root of your project:
 
+```json
+{
+    "presets": ["es2015"]
+}
+```
 
-## Setup
+If you do not add a `.babelrc` file (you may be writing es5 code for example) then the code will be bundled up as is.
 
-To integrate these tasks into your project you need to complete the following steps:
+#### JavaScript Linting
 
-Add `@justeat/gulp-build-fozzie` to your project
+Add an `.eslintrc` file to the root of your project which the following content to use the JS linting rules we recommend when running the [`scripts:lint`](#scriptslint) task:
 
-```bash
-yarn add @justeat/gulp-build-fozzie
+```json
+{
+    "extends": "@justeat/eslint-config-fozzie"
+}
 ```
 
-Inside your gulpfile require and then run `@justeat/gulp-build-fozzie` passing `gulp` as the first argument, You can optionally [pass in options here which will override the defaults](#options).
+#### CSS Linting
 
-```js
-const gulp = require('gulp');
-const build = require('@justeat/gulp-build-fozzie');
+To use our recommended fozzie stylelint linting rules, add the following into your `package.json` file:
 
-build(gulp, /*options*/);
+```json
+"stylelint": {
+    "extends": "@justeat/stylelint-config-fozzie"
+}
 ```
 
-That's it! You can now run any of [the Gulp tasks](#the-gulp-tasks).
+If you wish to extend these rules, you can also define your own `.stylelintrc` file in the root of your project with the rules that you wish to override.
```

Comment:
This is a alternate way of specifying the config.  You can extend the rules in the `package.json` too (like [here on uk ConsumerWeb](https://github.je-labs.com/CWA/ConsumerWeb/blob/master/src/ConsumerWeb/package.json#L50)).

Probably worth [linking to this](https://stylelint.io/user-guide/configuration/) as this defines the order of priority in whcih stylelint looks for config.